From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 9 Aug 2025 19:38:26 +0200
Subject: search: Link statically
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Otherwise searchd would require an additional shared
library.

Signed-off-by: Guido Günther <agx@sigxcpu.org>
Part-of: <https://gitlab.gnome.org/World/Phosh/phosh/-/merge_requests/1755>
(cherry picked from commit 06037ef8b97f9c31bda158eaafcde15d13f6222b)
---
 src/search/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/search/meson.build b/src/search/meson.build
index e25e1a9..585bea1 100644
--- a/src/search/meson.build
+++ b/src/search/meson.build
@@ -13,7 +13,7 @@ phoshsearch_sources = [
 
 phosh_search_deps = [gio_dep, gio_unix_dep, dependency('gdk-pixbuf-2.0')]
 
-phosh_search_lib = library(
+phosh_search_lib = static_library(
   'phoshsearch',
   phoshsearch_sources,
   include_directories: [root_inc, dbus_inc],
